ush/python/pygfs/task/stage_ic.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Any, Dict, List
+
+from wxflow import (AttrDict, FileHandler, Task, cast_strdict_as_dtypedict,
+                    logit, parse_j2yaml, strftime, to_YMD,
+                    add_to_datetime, to_timedelta, Template, TemplateConstants)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class Stage(Task):
+    """Task to stage initial conditions
+    """
+
+    @logit(logger, name="Stage")
+    def __init__(self, config: Dict[str, Any]) -> None:
+        """Constructor for the Stage task
+        The constructor is responsible for collecting necessary settings based on
+        the runtime options and RUN.
+
+        Parameters
+        ----------
+        config : Dict[str, Any]
+            Incoming configuration for the task from the environment
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(config)
+
+        self.task_config = AttrDict(**self.task_config)
```

Comment:
This line is redundant as it assigns `self.task_config` to itself.

Reply:
Will remove, thanks!

Reply:
Removed @ 30226501